extensions/amp-apester-media/0.1/amp-apester-media.js

```diff
@@ -109,6 +108,9 @@ class AmpApesterMedia extends AMP.BaseElement {
 
   /** @override */
   viewportCallback(inViewport) {
+    if (this.intersectionObserverApi_) {
```

Comment:
in addition to this, you also need to add
```
    if (this.intersectionObserverApi_) {
      this.intersectionObserverApi_.fire();
    }
```
to ```onLayoutMeasure```

Reply:
Thanks, Adding it up